e3sm_diags/driver/utils/dataset.py

```diff
@@ -179,6 +179,14 @@ def get_static_variable(self, static_var, primary_var):
             # Get the test variable from timeseries files.
             data_path = self.parameters.test_data_path
         file_path = self._get_timeseries_file_path(primary_var, data_path)
+        # Works:
+        #file_path=/lcrc/group/e3sm/public_html/e3sm_diags_test_data/unit_test_complete_run/test/time-series/streamflow_ts_postprocessed/RIVER_DISCHARGE_OVER_LAND_LIQ_198001_201412.nc
+        #static_var=areatotal2
+        # Fails:
+        #file_path=rof_links/RIVER_DISCHARGE_OVER_LAND_LIQ_198501_201412.xml
+        #static_var=areatotal2
+        print("file_path={}".format(file_path))
+        print("static_var={}".format(static_var))
```

Comment:
@chengzhuzhang It seems like the issue is `cdms2.open` can't read a `xml` file. So do we need to change what E3SM Diagnostics uses or do we need to make the fix in CDMS?

Reply:
It is weird, cdms2.open does open xml files. You could try on other single variable .xml files. I guess the problem is if cdms2.open reads a xml file with multiple variables.

Reply:
actually it work as follows:
```
file = 'RIVER_DISCHARGE_OVER_LAND_LIQ_198501_201412.xml'
import cdms2
fin = cdms2.open(file)
var = fin('RIVER_DISCHARGE_OVER_LAND_LIQ')
var = fin('areatotal2')
```